lib/renderer/init.ts

```diff
@@ -191,5 +174,5 @@ for (const preloadScript of preloadScripts) {
 // Warn about security issues
 if (process.isMainFrame) {
   const { securityWarnings } = require('@electron/internal/renderer/security-warnings');
-  securityWarnings(nodeIntegration);
+  securityWarnings(true);
```

Comment:
the `nodeIntegration` argument of `securityWarnings` is removed, you don't need to pass `true` here